recipe/meta.yaml

```diff
@@ -22,12 +22,23 @@ requirements:
     - pip
   run:
     - python >=3.10
-    - xarray >=2024.10.0
+    - xarray >=2025.1.1
     - numpy >=2.0.0
     - packaging
     - universal_pathlib
     - numcodecs
     - ujson
+  run_constrained:
+    - fsspec
+    - s3fs
+    - h5py
+    - hdf5
+    - hdf5plugin
+    - imagecodecs >=2024.6.1
+    - kerchunk >=0.2.8
+    - scipy
+    - astropy
+    - icechunk >=0.1.1
```

Comment:
FYI IIUC this section adds constraints for optional dependencies, but does not install them automatically.